.github/workflows/ci.yml

```diff
@@ -65,11 +65,6 @@ jobs:
             os: ubuntu-latest
             toxenv: py39-astropylts
 
-          - name: Twine check
-            python-version: "3.9"
-            os: ubuntu-latest
-            toxenv: twine
```

Comment:
Probably don't need it in tox.ini either. Up to you...

Reply:
I'll leave it in tox. Theoretically it would let us test it locally.